### PR TITLE
LIP-4 72h Aragon votes

### PR DESCRIPTION
**Pull request by kadmil**
The proposal to increase Aragon voting time from the current 24h to 72h.

### CODE REVIEW

File: LIPS/lip-4.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+status: WIP
+author: Victor Suzdalev (kadmil)
+discussions-to: https://research.lido.fi/t/increase-the-dao-voting-duration/1048
+created: 2021-09-28
+updated: -
+---
+
+# 72h Aragon Votes
+
+Currently Aragon votes on Lido DAO are 24h long. This timeframe has proved to be short enough for reaching the quorum even on non-contentios votes. While most routine DAO operations are expected to switch to EasyTracks (vehicle for pre-approved opt-out DAO actions), any custom blockchain action on behalf of DAO would still require the Aragon vote. The proposal is to make Aragon votes longer, allowing more LDO holders to participate.
```
Comment (vshvsh) [inline LIPS/lip-4.md]: > This timeframe has proved to be short enough for reaching the quorum even on non-contentios votes. 

It's harder to reach quorum on contentious votes, generally (aragon quorum is % of votes yes)

Comment (vshvsh) [reply LIPS/lip-4.md]: > pre-approved opt-out

default-pass

File: LIPS/lip-4.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+## Problem
+
+Gathering the quorum on Aragon votes within 24h window proved difficult enough even for repetitive and non-contentios actions such as raising Node Operators' keys limits and refilling liquidity pools' rewards programs.
+
+## Proposed solution
+
+The proposed solution is to make Aragon votes last 72h instead of the current 24h. To achieve this, the DAO would have to upgrade the Voting smart contract, as the current version doesn't allow changing the vote duration.
+
+The Voting contract version where the vote duration can be changed can be found [there](https://github.com/lidofinance/aragon-apps/blob/8c46da8704d0011c42ece2896dbf4aeee069b84a/apps/voting/contracts/Voting.sol)
```
Comment (vshvsh) [inline LIPS/lip-4.md:23]: +audit link

File: LIPS/lip-4.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+updated: -
+---
+
+# 72h Aragon Votes
+
+Currently Aragon votes on Lido DAO are 24h long. This timeframe has proved to be short enough for reaching the quorum even on non-contentios votes. While most routine DAO operations are expected to switch to EasyTracks (vehicle for pre-approved opt-out DAO actions), any custom blockchain action on behalf of DAO would still require the Aragon vote. The proposal is to make Aragon votes longer, allowing more LDO holders to participate.
+
+## Problem
+
+Gathering the quorum on Aragon votes within 24h window proved difficult enough even for repetitive and non-contentios actions such as raising Node Operators' keys limits and refilling liquidity pools' rewards programs.
```
Comment (vshvsh) [inline LIPS/lip-4.md]: > repetitive and non-contentios actions

non-contentious executive votes

File: LIPS/lip-4.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+   4. Call `unsafelyChangeVoteTime` method changing vote duration to 72h instead of the current 24h.
+3. Once the vote is concluded, enact it and get the 72h votes going forward.
+
+## Change caveats
+
+Any votes started within 48h before the voting upgrade vote start and not enacted before it's enactment would get their voting time frame updated as well. To prevent this side-effect, the voting upgrade must be started once all the running votes are concluded & enacted.
+For instance, on a week with single Omnibus vote starting on Thursday, 12PM UTC, the Omnibus should be enacted before starting the vote for Voting upgrade, and the Voting upgrade should start not earlier than 12PM UTC Saturday.
+
+It seems like the best time to start the Voting upgrade vote would be Monday, 11AM UTC. This would allow preserving Omnibus enactment schedule and bring as little distraction to DAO operations as possible.
+
```
Comment (vshvsh) [inline LIPS/lip-4.md:45]:
```suggestion

The increased vote time will reflect in all the activites that currently have a vote as a part of process, e.g. incentivization, changing limits etc.
```

File: LIPS/lip-4.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+The Voting contract version where the vote duration can be changed can be found [there](https://github.com/lidofinance/aragon-apps/blob/8c46da8704d0011c42ece2896dbf4aeee069b84a/apps/voting/contracts/Voting.sol)
+
+We've strived to make as little changes as possible so not to introduce any issues to the Voting contract the DAO currently uses.
+
+## Change implementation
+
+The upgrade plan is:
+1. Deploy the Voting implementation from [https://github.com/lidofinance/aragon-apps/blob/8c46da8704d0011c42ece2896dbf4aeee069b84a/apps/voting/contracts/Voting.sol](https://github.com/lidofinance/aragon-apps/blob/8c46da8704d0011c42ece2896dbf4aeee069b84a/apps/voting/contracts/Voting.sol).
+2. Run the Aragon vote with actions:
+   1. Push new app version to the Aragon repo
```
Comment (vshvsh) [inline LIPS/lip-4.md]: a bit more color here - you mean aragon pm? what's aragon repo?

**Review by vshvsh — CHANGES_REQUESTED** [review]

File: LIPS/lip-4.md (hunk 2 above)
Comment (kadmil) [reply LIPS/lip-4.md:23]: We wanted to share the LIP with auditors first, and will add it there after the audit is published

File: LIPS/lip-4.md (hunk 1 above)
Comment (kadmil) [reply LIPS/lip-4.md]: the point is "even on non-contentions votes reaching the quorum is hard enough"

**Comment by kadmil** [comment]
@vshvsh incorporated the feedback

File: LIPS/lip-4.md (hunk 5 above)
Comment (kadmil) [reply LIPS/lip-4.md]: Marked that it's our own fork of Aragon repo

**Pull request merged**